layouts/partials/feed-back.html

```diff
@@ -0,0 +1,8 @@
+<a class="edit-btn" target="_blank"
+  href='https://github.com/dgraph-io/dgraph-docs/issues/new'>
+  <i class="fa fa-thumbs-up"></i> Yes! content was helpful
+</a>
+<a class="edit-btn" target="_blank"
+  href='https://github.com/dgraph-io/dgraph-docs/issues/new'>
+  <i class="fa fa-thumbs-up"></i> No! content was not helpful
+</a>
```

Comment:
Can you add an extra line in the end? EOF.

Reply:
done!!!